cdc/log.cc

```diff
@@ -264,6 +266,13 @@ class cdc::cdc_service::impl : service::migration_listener::empty_listener {
                     schema.ks_name(), logname));
         }
     }
+
+    static void check_that_table_has_no_counter_columns(const schema& schema) {
```

Comment:
Nit beyond nit, I would suggest "ensure" instead of "check", since the latter should be stuff that checks and returns bool. While ensure is more in line with "assert"/throw.

Reply:
done!